pyproblems/factorial_number.py

```diff
@@ -0,0 +1,14 @@
+''' This program returns the factorial of the number passes'''
+from pyproblems.utility import is_int
+def factorial_num(number):
+    '''The function will return the factorial of the given number'''
+    if not is_int(number):
+        raise TypeError(f'unsupported format, {type(number)} is in the place of"integer"')
+    if number < 0:
+        raise ValueError("pass a positive integer")
+    if number == 1 or 0:
```

Comment:
Is this condition correct?

Reply:
Yes sir, the condition is correct and it yields the desired output.

Reply:
for the above condition. whether the "if" block execute if number is zero?